crates/optimism/node/src/engine.rs

```diff
@@ -147,6 +149,140 @@ where
             ))
         }
 
+        if self.chain_spec.is_fork_active_at_timestamp(
+            OptimismHardfork::Holocene,
+            attributes.payload_attributes.timestamp,
+        ) {
+            let Some(eip_1559_params) = attributes.eip_1559_params else {
+                return Err(EngineObjectValidationError::InvalidParams(
+                    "MissingEip1559ParamsInPayloadAttributes".to_string().into(),
+                ))
+            };
+            let (elasticity, denominator) = decode_eip_1559_params(eip_1559_params)
+                .map_err(|e| EngineObjectValidationError::InvalidParams(e.to_string().into()))?;
+            if elasticity != 0 && denominator == 0 {
```

Comment:
I don't see test coverage of this line

Reply:
https://github.com/paradigmxyz/reth/pull/11887/files#diff-c61b1dc2b86be8360bc0819742ae3d7a71810827532b244b5c05f5ff62a1f370R251

this is the test that tests 0 denominator case